submodules/javascript-functions/general.ts

```diff
@@ -173,4 +173,8 @@ export function countOccurrences(str: string, search: string): number {
         else break;
     }
     return c;
-} 
\ No newline at end of file
+}
+
+export function filterArrayMinusExcluded(array: any[], excludedArray: any[]) {
```

Comment:
Maybe just personal preference but for me this name was confusing because I thought it will filter values out of an array and skips some excluded ones from filtering. I would have used a signature e.g.

`export function filterArray(array: any[], filterValues: any[]) {`

Therefore it would be clear for me that I will filter the values out of the array. But just as an idea, you can of course decide on what you prefer.

Reply:
I would that **filterArray** may be too general because the filter can be done on different conditions.
What do think about this:
`function removeArrayFromArray(mainArray: any[], arrayToRemove: any[])`
or
`function excludeArrayFromArray(mainArray: any[], arrayToExclude: any[])`

Reply:
I agree that my suggestion is pretty generic and maybe not optimal. Out of the two you proposed, I think the first is clearer to me since remove is common for me in contrast to exclude as a programming term. But still, this is kinda subjective, so, please decide on the option you like the most. It was important for me to give you a perspective on how I understood the signature of the method :)

Reply:
Okay, let's name it as the first suggestion

Reply:
fits fine :)